test/README.md

```diff
@@ -14,26 +15,83 @@ Use `go test`:
 go test -v ./pkg/...
 ```
 
-## Running end-to-end tests
+_By default `go test` will not run [the e2e tests](#running-end-to-end-tests), which need [`-tags=e2e`](#running-end-to-end-tests) to be enabled._
 
-### Dependencies
+## Presubmit tests
 
-There's couple of things you need to install before running e2e tests locally.
+[`presubmit-tests.sh`](./presubmit-tests.sh) is the entry point for the [end-to-end tests](/test/e2e).
+
+This script, and consequently, the e2e tests will be run before every code submission. You can run these tests manually with:
 
 ```shell
+test/presubmit-tests.sh
+```
+
+_Note that to run `presubmit-tests.sh` or `e2e-tests.sh` scripts, you'll need kubernetes `kubetest` installed:_
+
+```bash
 go get -u k8s.io/test-infra/kubetest
 ```
 
-Simply run the `e2e-tests.sh` script. It will create a GKE cluster, install Knative Serving
-stack with Istio and run the end-to-end tests against the Knative Eventing built from source.
+## Running end-to-end tests
+
+To run [the e2e tests](./e2e) and [the conformance tests](./conformance), you need to have a running environment that meets
```

Comment:
There are no conformance tests for eventing yet...

Reply:
OK.